### PR TITLE
Added missing code for polymer model for output

### PR DESCRIPTION
**Pull request by babrodtk**
This fixes some code that was missing in #805 in the Polymer model

### CODE REVIEW

File: opm/polymer/fullyimplicit/BlackoilPolymerModel_impl.hpp [hunk 1]
```diff
@@ -598,7 +603,7 @@ namespace Opm {
 
         const ADB tr_mult = transMult(state.pressure);
         const ADB mu    = fluidViscosity(canonicalPhaseIdx, phasePressure[canonicalPhaseIdx], state.temperature, state.rs, state.rv, cond);
-        sd_.rq[phase].mob = tr_mult * kr[canonicalPhaseIdx] / mu;
+        sd_.rq[phase].mob = tr_mult * rq[phase].kr / mu;
```
Comment (babrodtk) [inline opm/polymer/fullyimplicit/BlackoilPolymerModel_impl.hpp]: ReservoirResidualQuant holds only active phases, and not all canonical phases (see line ~295 of this file)

Comment (atgeirr) [inline opm/polymer/fullyimplicit/BlackoilPolymerModel_impl.hpp]: Would it not be simpler to use `sd_.rq` here and not pass `rq` as a function argument?

File: opm/polymer/fullyimplicit/BlackoilPolymerModel_impl.hpp [hunk 2]
```diff
@@ -291,14 +291,19 @@ namespace Opm {
         // Set up the common parts of the mass balance equations
         // for each active phase.
         const V transi = subset(geo_.transmissibility(), ops_.internal_faces);
-        const std::vector<ADB> kr = computeRelPerm(state);
+        {
+            const std::vector<ADB> kr = computeRelPerm(state);
+            for (int phaseIdx=0; phaseIdx < fluid_.numPhases(); ++phaseIdx) {
```
Comment (atgeirr) [inline opm/polymer/fullyimplicit/BlackoilPolymerModel_impl.hpp]: Nitpick: always spaces around `=`.

File: opm/polymer/fullyimplicit/BlackoilPolymerModel_impl.hpp (hunk 1 above)
Comment (babrodtk) [inline opm/polymer/fullyimplicit/BlackoilPolymerModel_impl.hpp]: It appears so, yes. Thanks for spotting that!

**Pull request merged**